.github/workflows/dev-release.yml

```diff
@@ -0,0 +1,168 @@
+name: Dev Build
+on:
+  # Enable manually triggering this workflow via the API or web UI
+  workflow_dispatch:
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  test:
```

Comment:
I think you can safely remove `test` from here :thinking: No point in having it just waste time, right?